src/api/app/helpers/webui/buildresult_helper.rb

```diff
@@ -29,6 +29,7 @@ def arch_repo_table_cell(repo, arch, package_name, status = nil, enable_help = t
     end
   end
 
+  # NOTE: There is a JavaScript version of this method in project_monitor.js
```

Comment:
This comment is here because we expect that the other code using this method (`build_status`) will move to use Javascript in the future. Then the already implemented method can be used.